docs/pages/demo/daterangepicker/CalendarsDateRangePicker.example.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import Grid from '@material-ui/core/Grid';
-import Typography from '@material-ui/core/Typography';
+import { Typography, TextField } from '@material-ui/core';
```

Comment:
Did we settle on this approach?

```suggestion
import TextField from '@material-ui/core/TextField';
import Typography from '@material-ui/core/Typography';
```

Reply:
Let's make all these canges separately? Just a part of #1657

Reply:
Sure 💯

Reply:
My concern was with https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html, but addressing it, later on, is fine too.